config/src/config/indexer_grpc_config.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Default, Deserialize, PartialEq, Eq, Serialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct IndexerGrpcConfig {
+    pub enabled: bool,
+    pub address: String,
```

Comment:
shouldn't this be a url or something?

Reply:
could we get an example or some descriptions here?

Reply:
Also having a default value might be nice here, so we generally end up running this on the same port.

Reply:
Sure we can add a default for address. Just following the API pattern here.